packages/playwright-core/src/client/network.ts

```diff
@@ -645,6 +645,7 @@ export class Response extends ChannelOwner<channels.ResponseChannel> implements
 
   constructor(parent: ChannelOwner, type: string, guid: string, initializer: channels.ResponseInitializer) {
     super(parent, type, guid, initializer);
+    this.markAsInternalType();
```

Comment:
`class Request` as well maybe?

Reply:
good idea!

Reply:
Maybe we should start annotating getters in the protocol since it also applies to e.g. innerText etc.